csharp/OnnxRuntime.CSharp.sln

```diff
@@ -52,4 +51,4 @@ Global
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {C3DBDA2B-F169-4EDE-9353-858904124B75}
 	EndGlobalSection
-EndGlobal
```

Comment:
>ndGlobal [](start = 1, length = 8)

This file does not seem to have any changes

Reply:
Yes, I tried getting it back to its old shape but git keeps detecting the single line change at the top. Not sure what is happening.